src/leo-client-app/package.json

```diff
@@ -22,7 +22,9 @@
     "moment": "^2.29.4",
     "next": "^13.5.4",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "react-router-dom": "^6.21.3",
+    "terminal-kit": "^3.0.1"
```

Comment:
move all the terminal stuff to another branch/pr, no point having it with schedule ui changes

Reply:
also why are you using react-dom/react-router-dom here? the entire project uses next